docs/source/concept_guides/performance.mdx

```diff
@@ -89,3 +89,9 @@ learning_rate *= accelerator.num_processes
 optimizer = AdamW(params=model.parameters(), lr=learning_rate)
 ```
 
+You will also find that `accelerate` will step the learning rate based upon the number of processes, so if you have a learning rate scheduler
+that steps every 1000 steps, it will step every 1000 * `num_processes` steps. You should adjust the number of steps
```

Comment:
No there is no adjusting to be done that is the whole point. This is to account for the fact the dataloader length has been divided by `num_processes`.

Reply:
@sgugger I changed this to:

You will also find that `accelerate` will step the learning rate based on the number of processes being trained on. This is because 
of the observed batch size noted earlier. So in a case of 2 GPUs, the learning rate will be stepped twice as often as a single GPU
to account for the batch size being twice as large (if no changes to the batch size on the single GPU instance are made).

That better? 😄